samples/highcharts/yaxis/stackshadow/demo.js

```diff
@@ -0,0 +1,75 @@
+Highcharts.chart('container', {
+    chart: {
+        type: 'pictorial'
+    },
+
+    title: {
+        text: 'Composition of the human body'
+    },
+
+    xAxis: {
+        categories: ['Woman', 'Man'],
+        lineWidth: 0,
+        opposite: true
+    },
+
+    yAxis: {
+        visible: false,
+        stackShadow: {
+            enabled: true
+        },
+        max: 100
```

Comment:
Maybe it's more semantically correct to set `series.stacking` to `percent`, and then we won't have to set `yAxis.max`.

Reply:
The main reason why is that, when stacking is percent, the `stackShadow` is never visible. In my opinion, this demo looks even better with normal stacking when playing around. But if you think it's not quite logical, I can change it to percent and create a separate demo for `stackShadow` option. Let me know what you think 🤔

Reply:
Ah, yes I see that now. If we set stacking to percent, the data will fill the view no matter what. Let's keep this as it is then.